### ISSUE TITLE
Incompatability with skript-gui

### ISSUE THREAD

**Issue by Netherpwn** [aabssmc/Skuishy#7]
The addon somehow mess with skript-gui specifically when setting items that has different colors/variations. 

To replicate:

1. Use Skript, skript-gui, Skuishy in a Purpur 1.20.4 server.
2. Create a skript file and copy the code snippet from skript-gui to create a quick gui. In this case, I am using the 3rd code snippet then replaced `dirt` with `black stained glass pane` from this link: https://github.com/APickledWalrus/skript-gui/wiki/2.-Modifying-the-contents-of-a-GUI
```
command /opengui:
  trigger:
    create a gui with virtual chest inventory with 3 rows named "My EPIC GUI!" and shape "xxxxxxxxx", "x-------x", and "xxxxxxxxx":
      make gui slot "x" with black stained glass pane named "! BORDER !"
    open the last gui for the player
```

3. Save the skript file then restart the server. Do the command `/opengui` multiple times and you'll notice that the panes will change color as if I set the item to any stained glass pane.

Hopefully you will fix this. Great addon by the way.

**Comment by ghost** [comment]
what skuishy and skript version you on

**Comment by Netherpwn** [comment]
Oh it's fixed in 2.4, sorry.

**Comment by ghost** [comment]
your good
